### DISCUSSION TITLE
Hiding bufferline on the Dashboard?

### DISCUSSION THREAD

**Discussion by bitzcoder** [LazyVim/LazyVim#4946] [Q&A] [answered]
I have configured `bufferline.nvim` to always show the bufferline, even when there is only one buffer open. This is important for me to see if the buffer is modified.

```lua
  {
    "akinsho/bufferline.nvim",
    opts = function(_, opts)
      opts.options.always_show_bufferline = true
    end,
  },

```
However, this causes the bufferline to appear even on the dashboard. **Is there a way to hide the bufferline only when the dashboard is displayed?**

I tried using an autocmd to hide the tabline when the dashboard opens and restore it when exiting, but it doesn’t seem to work as expected:

```lua
vim.api.nvim_create_autocmd("User", {
  pattern = "SnacksDashboardOpened",
  callback = function()
    vim.opt.showtabline = 0
    vim.api.nvim_create_autocmd("BufUnload", {
      buffer = vim.api.nvim_get_current_buf(),
      callback = function()
        vim.opt.showtabline = 2
      end,
    })
  end,
})
```
TIA.

**Comment by dpetka2001** [comment, marked as answer]
```lua
  {
    "akinsho/bufferline.nvim",
    opts = {
      options = {
        auto_toggle_bufferline = false,
      },
    },
  },
```
```lua
vim.api.nvim_create_autocmd("BufRead", {
  group = vim.api.nvim_create_augroup("test", {}),
  desc = "test",
  callback = function()
    vim.opt.showtabline = 2
  end,
})
```
Bufferline is weird. And I'm not sure if other edge cases might arise from this. Might not be 100% solution, but try it out and see if it fits your needs.
